server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/GroupWsRef.java

```diff
@@ -69,7 +71,7 @@ public String getOrganizationUuid() {
   }
 
   /**
-   * @return the non-null group name.
+   * @return the non-null group name. Can be anyone.
```

Comment:
missing quote around `anyone` :)

We can't tell right now whether it the returned value you talk about or that it can return anything